src/Compute/v2/Models/QuotaSet.php

```diff
@@ -0,0 +1,170 @@
+<?php declare(strict_types=1);
+
+namespace OpenStack\Compute\v2\Models;
+
+use OpenStack\Common\Resource\Deletable;
+use OpenStack\Common\Resource\OperatorResource;
+use OpenStack\Common\Resource\Retrievable;
+use OpenStack\Common\Resource\Updateable;
+
+/**
+ * Represents a Compute v2 Quota Set
+ *
+ * @property \OpenStack\Compute\v2\Api $api
+ */
+class QuotaSet extends OperatorResource implements Retrievable, Updateable, Deletable
+{
+    /**
+     * The number of allowed instance cores for each tenant.
+     *
+     * @var  int|array
```

Comment:
We certainly can implement that way, but then at the same time I don't want the SDK to diverse from the API response.